cli/src/main/resources/help/credentials.adoc

```diff
@@ -0,0 +1,72 @@
+CREDENTIALS(1)
+==============
+:doctype: manpage
+
+
+NAME
+----
+credentials - manages ${infinispan.brand.name} credential stores
+
+
+SYNOPSIS
+--------
+*credentials ls*
+
+*credentials create* 'alias'
+
+*credentials remove* 'alias'
+
+
+DESCRIPTION
+-----------
+Manage (list, create, remove) credentials within a key store. Unless overridden, the commands use the `credentials.pfx`
+key store in the server's configuration directory.
+
+
+COMMAND SYNOPSIS
+----------------
+
+List credentials in a key store
+
+*credentials ls*::
+Lists the credentials stored in the key store.
+
+Create a credential
+
+*credentials create* 'alias'::
+Creates a credential with the specified alias (name) within the key store.
+
+Remove a credential
+
+*credentials remove* 'alias'::
+Removes the credential identified by the specified alias (name) from the key store.
+
+
+COMMON OPTIONS
+--------------
+
+These options apply to all commands:
+
+*-h, --help*::
+Displays a help page for the command or sub-command.
+
+*--server-root*='path-to-server-root'::
```

Comment:
is the `--server-root` missing from `Credentials` command?

Reply:
I've added it.